docs/document.rst

```diff
@@ -1220,22 +1221,24 @@ For details on **embedded files** refer to Appendix 3.
 
     :arg str user_pw: *(new in v1.16.0)* set the document's user password. The string length must not exceed 40 characters.
 
+    :arg int use_objstms: *(new in v1.24.0)* compression option that converts eligible PDF object definitions to information that is stored in some other object's :data:`stream` data. Depending on the `deflate` parameter value, the converted object definitions will be compressed -- which can lead to very significant file size reductions.
+
     .. warning:: The method does not check, whether a file of that name already exists, will hence not ask for confirmation, and overwrite the file. It is your responsibility as a programmer to handle this.
 
     .. note::
 
       **File size reduction**
 
-      1. Use the adequate save options like `garbage=3` or `4`, `deflate=True`. Do not use "expand", "clean" or "incremental".
-      This is a "lossless" file size reduction.
+      1. Use the save options like `garbage=3|4, deflate=True, use_objstms=True|1`. Do not touch the default values `expand=False|0, clean=False|0, incremental=False|0`.
+      This is a "lossless" file size reduction. There is a convenience version of this method with these values set by default, :meth:`Document.ez_save` -- please see below. 
 
-      2. "Lossy" file size reduction in essence must give up something with respect to images, like (a) remove all images (b) replace images by their grayscale versions (c) reduce image resolutions.
+      2. "Lossy" file size reduction in essence must give up something with respect to images, like (a) remove all images (b) replace images by their grayscale versions (c) reduce image resolutions. Find example scripts `here <https://github.com/pymupdf/PyMuPDF-Utilities/tree/master/examples/replace-image>`_.
```

Comment:
Let's say: Find example scripts on our *PyMuPDF Utilities "replace image" example* ( italics are the hyperlink )